db/migrate/20200225170408_change_role_to_discipline.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeRoleToDiscipline < ActiveRecord::Migration[6.0]
```

Comment:
It would have been better to call out that this affects the team member table like you have for other migrations. That makes it easier to see at a glance what was affected just by the file / class name.